### DISCUSSION TITLE
[2021-10-21] 중복 제거하기

### DISCUSSION THREAD

**Discussion by hyk339** [pyunji/Hyundai-ITnE-Team_QuadCore#153] [SQL]
[중복 제거하기](https://programmers.co.kr/learn/courses/30/lessons/59408)

**Comment by pyunji** [comment]
[중복 제거하기 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/sql/PR_59408.sql)

**Comment by chaeyeon4u** [comment]
[중복 제거하기 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/SQL/PR_59408.sql)
